judgeops/Readme.md

```diff
@@ -1,114 +1,18 @@
-## Guide to use `judgeops`
+1)  `./judgeops -m db.json apply-judgement tests/post_audit.report-config.judge.md`
```

Comment:
Do these steps apply in the same way to a `nyth` project? 👀

Reply:
💯 Absolutely